app/_src/policies/meshtcproute.md

```diff
@@ -0,0 +1,259 @@
+---
+title: MeshTCPRoute (beta)
+---
+
+{% warning %}
+This policy uses new policy matching algorithm and is in beta state,
+it should not be mixed with [TrafficRoute](../traffic-route).
+{% endwarning %}
+
+The `MeshTCPRoute` policy allows altering and redirecting TCP requests
+depending on where the request coming from and where it's going to.
+
+## TargetRef support matrix
+
+| TargetRef type    | top level | to | from |
+|-------------------|-----------|----|------|
+| Mesh              | ✅         | ❌  | ❌    |
+| MeshSubset        | ✅         | ❌  | ❌    |
+| MeshService       | ✅         | ✅  | ❌    |
+| MeshServiceSubset | ✅         | ❌  | ❌    |
+| MeshGatewayRoute  | ❌         | ❌  | ❌    |
+
+If you don't understand this table you should read 
+[matching docs](/docs/{{ page.version }}/policies/targetref).
+
+## Configuration
+
+Unlike others outbound policies `MeshTCPRoute` doesn't contain `default`
+directly in the `to` array. The `default` section is nested inside`rules`,
+so the policy structure looks like this:
+
+```yaml
+spec:
+  targetRef: # top-level targetRef selects a group of proxies to configure
+    kind: Mesh|MeshSubset|MeshService|MeshServiceSubset 
+  to:
+    - targetRef: # targetRef selects a destination (outbound listener)
+        kind: MeshService
+        name: backend
+      rules:
+        - default: # configuration applied for the matched TCP traffic
+            backendRefs: [...]
+```
+
+### Default conf
+
+- **`backendRefs`** - (optional) - list of destination for request to be
+  redirected to
+  - **`kind`** - one of `MeshService`, `MeshServiceSubset`
+  - **`name`** - service name
+  - **`tags`** - service tags, must be specified if the `kind` is 
+    `MeshServiceSubset`
+  - **`weight`** - when a request matches the route, the choice of an upstream
+    cluster is determined by its weight. Total weight is a sum of all weights
+    in `backendRefs` list.
+
+
+## Examples
+
+### Traffic split
+
+We can use `MeshTCPRoute` to split a TCP traffic between services with
+different tags implementing A/B testing or canary deployments.
```

Comment:
```suggestion
You can use `MeshTCPRoute` to split TCP traffic between services with
different tags and implement A/B testing or canary deployments.
```

Reply:
We already have `vale` set up, it may have broken at some point. I'll investigate it

Reply:
@slonka @michaelbeaumont I'm not sure if Vale is checking for "we", but that's a really good suggestion. To get that added, the docs team might have to open a PR to add that rule to the Vale open source repo.